crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -415,9 +420,7 @@ impl Room {
     #[instrument(skip_all, fields(room_id = ?self.room_id))]
     pub async fn is_direct(&self) -> StoreResult<bool> {
         match self.state() {
-            RoomState::Joined | RoomState::Left => {
```

Comment:
Looks like nothing changed here - did you mean to do this, or did your editor get clever on you?

Reply:
Maybe, but I think I just run the fixup style task and it re-formatted the code here after removing the `RoomState::` prefix, then adding it back.